cardano-node/docs/tx-submit-using-tx-gen.md

```diff
@@ -0,0 +1,96 @@
+# Submit Transactions to WebAPI using Transaction Generator
+
+This document demonstrates how to submit transactions to `cardano-tx-submit-webapi` using [transaction generator](https://github.com/input-output-hk/cardano-node/blob/master/cardano-node/src/Cardano/CLI/Tx/Generation.hs).
+
+## Prepare Local Cluster to Start
+
+Go to `cardano-node` repository and remove nodes' local databases if needed:
+
+```
+$ rm -rf db/
+```
+
+Also remove an old genesis files:
+
+```
+$ rm -rf configuration/genesis/*
+```
+
+Now create the new genesis:
+
+```
+$ ./scripts/genesis.sh
+```
+
+Please see an output, you should see something like this:
+
+```
+...
+genesis created with hash = b8f44e852982a88aaf352684bc507fd337a15117f01f43816abfb2487a67d348
+...
+```
+
+## Run Local Cluster
+
+Go to `tmux` and launch the local cluster (Shelley in Live View mode):
+
+```
+$ tmux
+$ ./scripts/shelley-testnet-live.sh
+```
+
+You will see 3 `tmux` panes with the nodes. Please see the `block number` value: if it increases, the cluster is working correctly.
+
+## Run Tx Submit WebAPI
+
+Go to `cardano-explorer` repository and create configuration for `cardano-tx-submit-webapi`:
+
+```
+$ ./scripts/gen-tx-submit-config.sh --require-magic --genesis-hash GEN_HASH --output config.yaml
+```
+
+where `GEN_HASH` is the hash of genesis you've got from the output of `./scripts/genesis.sh` script. As a result, you will see the file `config.yaml` in the current directory.
+
+Run `cardano-tx-submit-webapi`:
+
+```
+$ cabal new-run cardano-tx-submit-webapi -- --config config.yaml --genesis-file /path-to/cardano-node/configuration/genesis/genesis.json --socket-path /path-to/cardano-node/socket/0 --port 8101
+```
+
+Possible output is this:
+
+```
+[cardano-tx-submit:Info:9] [2020-02-11 08:05:34.82 UTC] Running tx-submit node
+[cardano-tx-submit:Info:9] [2020-02-11 08:05:34.82 UTC] NetworkMagic: RequiresMagic 459045235
+[cardano-tx-submit:Info:9] [2020-02-11 08:05:34.82 UTC] localInitiatorNetworkApplication: connecting to node via "/home/shevchenko/Code/cardano-node/socket/0"
```

Comment:
minor: you may want to remove references to your home directory with something like <...> or so.

Reply:
Fixed.